### PR TITLE
AC Automated Test - LRAC-8424 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8424

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 1]
```diff
@@ -118,11 +118,119 @@ definition {
...
+			ACSegments.saveSegment();
+		}
+
+		task ("Go to form and submit to get data for segmented individuals") {
+			ACUtils.launchDXP();
 
-		// TODO LRAC-8424 AudienceCardShowsExpectedAmountSegmentedAndUnsegmentedInForms
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "ac@liferay.com",
+				userLoginFullName = "ac ac");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase]: Hi @AfonsoHSantos, I think you could login that way `ACUtils.launchDXP(userEmailAddress = "ac@liferay.com");` and so avoid using the `User.logoutAndLoginPG` macro

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 2]
```diff
@@ -118,11 +118,119 @@ definition {
...
+
+			User.logoutPG();
+
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			FormFields.editText(
+				fieldName = "Text",
+				fieldValue = "Test Assets");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:206]: Hi @AfonsoHSantos, I believe that this visit to the form with an anonymous user could be retired. What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase (hunk 1 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase]: fixed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase (hunk 2 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase]: hi @ViniLopes20 , i do that. thank you!

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**